executor/builder.go

```diff
@@ -2314,7 +2314,6 @@ func (b *executorBuilder) buildTableReader(v *plannercore.PhysicalTableReader) *
 	}
 
 	ts := v.GetTableScan()
-	logutil.BgLogger().Info("build table reader", zap.Int64("my table id", ts.Table.ID), zap.String("table name", ts.Table.Name.L), zap.String("range str", ts.Ranges[0].String()))
```

Comment:
why delete this line?

Reply:
Some test forbid to output any log during `insert` and `update`, see https://github.com/pingcap/tidb/blob/82829e1fa5b908cf4b6ea8c97ba14a38fcde81fa/executor/seqtest/seq_executor_test.go#L1436-L1435